docs/sources/operator/api.md

```diff
@@ -340,7 +340,7 @@ MetricsInstanceSpec controls how an individual instance is used to discover PodM
 |`probeSelector`<br/>_[Kubernetes meta/v1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#labelselector-v1-meta)_|  ProbeSelector determines which Probes to select for target discovery.  |
 |`probeNamespaceSelector`<br/>_[Kubernetes meta/v1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#labelselector-v1-meta)_|  ProbeNamespaceSelector is the set of labels that determines which namespaces to watch for Probe discovery. If nil, it only checks own namespace.  |
 |`remoteWrite`<br/>_[[]RemoteWriteSpec](#monitoring.grafana.com/v1alpha1.RemoteWriteSpec)_|  RemoteWrite controls remote_write settings for this instance.  |
-|`additionalScrapeConfigs`<br/>_[Kubernetes core/v1.SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#secretkeyselector-v1-core)_|  AdditionalScrapeConfigs lets you specify a key of a Secret containing additional Grafana Agent Prometheus scrape configurations. The specified scrape configurations are appended to the configurations generated by Grafana Agent Operator. Specified job configurations must have the form specified in the official Prometheus documentation: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config. As scrape configs are appended, you must make sure the configuration is still valid. Note that it&#39;s possible that this feature will break future upgrades of Grafana Agent. Review both Grafana Agent and Prometheus release notes to ensure that no incompatible scrape configs will break Grafana Agent after the upgrade.  |
+|`additionalScrapeConfigs`<br/>_[Kubernetes core/v1.SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#secretkeyselector-v1-core)_|  AdditionalScrapeConfigs lets you specify a key of a Secret containing additional Grafana Agent Prometheus scrape configurations. The specified scrape configurations are appended to the configurations generated by Grafana Agent Operator. Specified job configurations must have the form specified in the official Prometheus documentation: https://prometheus.io/docs/prometheus/2.42/configuration/configuration/#scrape_config. As scrape configs are appended, you must make sure the configuration is still valid. Note that it&#39;s possible that this feature will break future upgrades of Grafana Agent. Review both Grafana Agent and Prometheus release notes to ensure that no incompatible scrape configs will break Grafana Agent after the upgrade.  |
```

Comment:
This file is automatically generated from code comments, it might get overwritten again when it's regenerated.

Reply:
What is the process for regenerating it? I could find it in the readme, and when I run `make docs` this file stays the same also.

Reply:
You may want to reach out to the team that manages the operator, because I'm not sure how it's rebuilt.

Reply:
I see, so this file is pretty much a copy of the one in the official Operator repo. I'll just exclude it from this PR.